src/axom/multimat/docs/sphinx/multimat_materials.rst

```diff
@@ -0,0 +1,119 @@
+use the MultiMat object to define materials.
+
+#######################
+Cell Material Relation
+#######################
+
+The distribution of materials over the mesh is determined using the **Cell-Material Relation** *(CMR)*.
+If a mesh has *N* cells and *M* materials, each cell can have up to *M* possible values.
+This means the mesh will hold N*M values if each cell contains all materials.
+There are multiple ways to specify the CMR to MultiMat. A static material decomposition
+is described here, though the CMR can also be built :doc:`dynamically <multimat_dynamic_mode>`.
+The easiest method for defining the CMR is to provide a bool vector containing
+true/false values for whether a cell/material combination is valid.
+
+.. figure:: figures/relation.png
+   :figwidth: 700px
+
+   Diagram showing mixed-material mesh with CELL_DOM and MAT_DOM ways of defining the Cell-Material Relation.
+
+The following code shows how to initialize a MultiMat object with 9 cells and 3 materials
+and build the CMR by populating a bool vector that is given to the MultiMat object.
+In the following example, the relation is expressed using a **Cell-Dominant**
+data layout (``CELL_DOM``). This means that the data will be arranged such that all material
+values for cell 0 are given, followed by all of the material values for cell 1, until all
+cells have provided their flags. The bool vector contains *true* if a material is present
+in a cell and *false* if the material is not present. This CMR vector is essentially
+a mask for which cell/material combinations are valid. Data can also be transposed
+into a **Material-Dominant** data layout (``MAT_DOM``) in which the materials are
+iterated first, followed by cells that use the current material.
+
+.. code-block:: cpp
+
+    constexpr int nmats = 3;
+    constexpr int ncells = 9;
+
+    // Create the MultiMat object
+    axom::multimat::MultiMat mm;
+    mm.setNumberOfMaterials(nmats);
+    mm.setNumberOfCells(ncells);
+
+    // Cell-Dominant data layout
+    int rel[9][3] = {{1,1,0},
+                     {1,1,1},
+                     {0,1,1},
+                     {1,1,1},
+                     {0,1,1},
+                     {0,0,1},
+                     {0,1,1},
+                     {0,0,1},
+                     {0,0,1}};
+    std::vector<bool> relation(ncells * nmats, false);
+    for(int c = 0; c < ncells; c++)
+    {
+      for(int m = 0; m < nmats; m++)
+      {
+        relation[c * nmats + m] = rel[c][m] > 0;
+      }
+    }
+    mm.setCellMatRel(relation, axom::multimat::DataLayout::CELL_DOM);
+
+
+#######################
+Volume Fractions
+#######################
+
+The CMR determines how materials are allocated to each cell but it does not determine
+how much of each material is present in the cell. Volume fractions determine how much of each material
+exists in each cell and is given as a percentage. If a cell contains materials A and B
```

Comment:
Please consider rephrasing to "The CMR determines which materials are present in each cell; volume fractions determine how much of  each material is present in each cell."  This shortens the first two sentences to one sentence with two parallel phrases: to me, this helps with clarity.  Also, I don't think you need "given as a percentage," since the next sentence gives an example.

Reply:
Changed.